### ISSUE TITLE
Things to do for Dakota's Evaluation suite [Oct 15]

### ISSUE THREAD

**Issue by tanzir5** [odissei-lifecourse/life-sequencing-dutch#94]
1. fix the "99999999999" issue
2. change the target column to "INPPERSPRIM" [@amacanovic  can confirm]
3. run it on the entire data. 
4. add an option to run it for just baseline (no embeddings)
5. fix prepare_naive_baseline.py

**Comment by tanzir5** [comment]
<img width="754" alt="Screenshot 2024-10-14 at 8 56 13 PM" src="https://github.com/user-attachments/assets/300b5ebd-4bff-4292-992b-b2ea40be7aa3">


Actually a combination of these two variables may be even better?

**Comment by dakota0064** [comment]
What do we mean by combination here, just a simple sum? It *should* be the case that most people only have one or the other, but I can imagine a world where somebody holds two jobs, one public and one private.

**Comment by amacanovic** [comment]
I agree we should add the two up; a simple sum will do. People will mostly either be a government/public sector employee (ambtenaar) or employed in a private sector, rarely both, but we want to capture both groups of people. The only reason I haven't done that was time constraints.
